recipes/sdl/all/conanfile.py

```diff
@@ -194,6 +197,11 @@ def _patch_sources(self):
     @functools.lru_cache(1)
     def _configure_cmake(self):
         cmake = CMake(self)
```

Comment:
```suggestion
        cmake = CMake(self)
        cmake.definitions["SDL2_DISABLE_INSTALL"] = False
```

Reply:
it still the same on latest release 11 days ago:
https://github.com/libsdl-org/SDL/blob/main/CMakeLists.txt#L425-L427

Reply:
> 

it still the same on latest release 11 days ago:
https://github.com/libsdl-org/SDL/blob/main/CMakeLists.txt#L425-L427

Reply:
You need to apply this patch.
Because of this:
```cmake
set_option(SDL2_DISABLE_INSTALL    "Disable installation of SDL2" ${SDL2_SUBPROJECT})
```